SampleApp/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SampleApp.Models;
+using SampleApp.Repository;
+
+namespace SampleApp.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Create a constant file and move the consts into that file. APi related things should go into file ApiRoutes